shared/fs/send-link-to-chat/index.js

```diff
@@ -58,110 +59,242 @@ const who = (props: Props) => {
+  ) : (
+    <Kb.Text type={Styles.isMobile ? 'BodySemibold' : 'Header'}>
+      Send link to{' '}
+      {props.conversation.type === 'small-team' || props.conversation.type === 'big-team'
+        ? 'team chat'
+        : 'group chat'}
+    </Kb.Text>
   )
 
-const Header = (props: Props) => (
-  <Kb.Box2 direction="horizontal" centerChildren={true} style={styles.header} fullWidth={true}>
-    {props.conversation.type === 'none' ? (
-      <Kb.Text type="Header">Copy link</Kb.Text>
-    ) : (
-      <>
-        <Kb.Text type="Header">Send Link to</Kb.Text>
-        <Kb.Box style={styles.headerGap} />
-        {(props.conversation.type === 'small-team' || props.conversation.type === 'big-team') && (
-          <Kb.Avatar size={16} teamname={props.conversation.name} isTeam={true} style={styles.avatar} />
-        )}
-        <CommaSeparatedName type="Header" name={props.conversation.name} />
-      </>
+const DesktopHeader = (props: Props) => (
+  <Kb.Box2 direction="horizontal" centerChildren={true} style={styles.desktopHeader} fullWidth={true}>
+    <HeaderContent {...props} />
+  </Kb.Box2>
+)
+
+const Footer = (props: Props) => (
+  <Kb.Box2 direction="horizontal" centerChildren={true} style={styles.footer} gap="tiny">
+    {!Styles.isMobile && <Kb.Button type="Secondary" label="Cancel" onClick={props.onCancel} />}
+    {props.conversation.type !== 'none' && (
+      <Kb.Button type="Primary" label="Send in conversation" disabled={!props.send} onClick={props.send} />
     )}
   </Kb.Box2>
 )
 
-const SendLinkToChat = (props: Props) => (
-  <Kb.Box2 direction="vertical" style={styles.container}>
-    <Header {...props} />
-    <Kb.Box2 direction="vertical" fullWidth={true} fullHeight={true} centerChildren={true}>
-      <Kb.CopyText text={props.pathTextToCopy} containerStyle={styles.copyText} />
-      {props.conversation.type !== 'none' && (
-        <Kb.Text type="BodyTiny" style={styles.onlyWhoGetAccess}>
-          Only {who(props)} will get access to the file.
-        </Kb.Text>
-      )}
-      <BigTeamChannelDropdown {...props} />
+const SendLinkToChatMain = (props: Props) => (
+  <Kb.Box2 direction="vertical" fullWidth={true} centerChildren={true} style={styles.main}>
+    <Kb.Box2 direction="horizontal" fullWidth={true} centerChildren={true} style={styles.centerBox}>
+      <Kb.CopyText text={props.pathTextToCopy} multiline={Styles.isMobile} />
     </Kb.Box2>
-    <Kb.Box2 direction="horizontal" centerChildren={true} style={styles.footer} gap="tiny">
-      <Kb.Button type="Secondary" label="Cancel" onClick={props.onCancel} />
-      {props.conversation.type !== 'none' && (
-        <Kb.Button type="Primary" label="Send in conversation" disabled={!props.send} onClick={props.send} />
-      )}
+    {props.conversation.type !== 'none' && (
+      <Kb.Text type="BodySmall" style={styles.onlyWhoGetAccess}>
+        Only {who(props)} will get access to the file.
```

Comment:
I was confused by this because "will get access" sounds like this grants them access, when they already have access. Could it change to "Only {...} has/have access to this file."?

Reply:
Sounds good with me. @cecileboucheron how does that sound?